pkg/collectors/contexts/contexts.go

```diff
@@ -5,14 +5,19 @@ package contexts
 import (
 	"fmt"
 
+	corev1 "k8s.io/api/core/v1"
+
 	"github.com/redhat-partner-solutions/vse-sync-collection-tools/pkg/clients"
 )
 
 const (
-	PTPNamespace     = "openshift-ptp"
-	PTPPodNamePrefix = "linuxptp-daemon-"
-	PTPContainer     = "linuxptp-daemon-container"
-	GPSContainer     = "gpsd"
+	PTPNamespace               = "openshift-ptp"
+	PTPPodNamePrefix           = "linuxptp-daemon-"
+	PTPContainer               = "linuxptp-daemon-container"
+	GPSContainer               = "gpsd"
+	NetlinkDebugPod            = "ptp-dpll-netlink-debug-pod"
+	NetlinkDebugContainer      = "ptp-dpll-netlink-debug-container"
+	NetlinkDebugContainerImage = "quay.io/jnunez/tools:dpll9.2_dev"
```

Comment:
I'm pushing for getting this debugging container in the ptp operator when running in debug mode but we are not there yet. I moved this container to redhat-partner-solutions instead of my quay account.
```suggestion
	NetlinkDebugContainerImage = "quay.io/redhat-partner-solutions/tools:dpll9.2_dev"
```

Reply:
@jnunyez Is this image available yet?

Reply:
No, not yet. I will ping you when it is there.

Reply:
@nocturnalastro give it a try it is also public for pulling and changed a little bit the nomenclature: 
https://quay.io/repository/redhat-partner-solutions/dpll-debug?tab=tags&tag=0.1